bin/kafka-run-class.sh

```diff
@@ -142,6 +142,13 @@ if [ -z "$KAFKA_JVM_PERFORMANCE_OPTS" ]; then
   KAFKA_JVM_PERFORMANCE_OPTS="-server -XX:+UseG1GC -XX:MaxGCPauseMillis=20 -XX:InitiatingHeapOccupancyPercent=35 -XX:+DisableExplicitGC -Djava.awt.headless=true"
 fi
 
+# version option
+for args in "$@" ; do
+  if [ "$args" == "--version" ]; then
```

Comment:
Within a `test` expression like this, it's discouraged to use `==`. A single equals tests if strings are identical:

```diff
-if [ "$args" == "--version" ]; then
+if [ "$args" = "--version" ]; then
```